src/utils/smartweave/tags.go

```diff
@@ -3,15 +3,18 @@ package smartweave
 import "regexp"
 
 const (
-	TagAppName         = "App-Name"
-	TagAppNameValue    = "SmartWeaveAction"
-	TagAppVersion      = "App-Version"
-	TagContractTxId    = "Contract"
-	TagInput           = "Input"
-	TagContentType     = "Content-Type"
-	TagContractSrcTxId = "Contract-Src"
-	TagSDK             = "SDK"
-	TagMinFee          = "Min-Fee"
+	TagAppName              = "App-Name"
+	TagAppNameValue         = "SmartWeaveAction"
+	TagAppVersion           = "App-Version"
+	TagContractTxId         = "Contract"
+	TagInput                = "Input"
+	TagInputFormat          = "Input-Format"
+	TagInputFormatTagValue  = "tag"
```

Comment:
Move to another const group, add a comment that those are values of the input-format tag